src/util/game-activity-tracker/statistics.ts

```diff
@@ -75,6 +80,29 @@ export async function statisticsGameStats(interaction: ChatInputCommandInteracti
 		.setTitle(`Stats across all users for ${game.replace(/(\b\w)/g, (e) => e.toUpperCase())}!`)
 		.addFields(fields)
 		.addFields({ name: "Users", value: `${users.length} unique gaymers :)`, inline: true });
+
+	if (showPlaytime) {
+		const entries = activityTrackerLogDb.filter((val, key) => key.split("-")[0] === game);
```

Comment:
We already have a variable called `entrys` above which contains the entries for the game, refetching like this will cause performance issues

Reply:
sadly it does not... `entrys` is just an array of the relevant keys...

Reply:
my entries also contains the values